hangups/ui/__main__.py

```diff
@@ -198,11 +198,23 @@ def _on_event(self, conv_event):
 
     def _on_quit(self):
         """Handle the user quitting the application."""
-        future = asyncio.async(self._client.disconnect())
-        future.add_done_callback(lambda future: future.result())
+        asyncio.ensure_future(self._client.disconnect())
```

Comment:
Ubuntu 14.04 is stuck on Python 3.4.3. I'm still using that so I'd like to keep it supported a bit longer.